app/styles/nypr-ui/_buttons.scss

```diff
@@ -109,39 +109,69 @@
 }
 
 /**
- * Blue Button:
- * `.btn--blue`
+ * Large Button:
+ * `.btn--large`
+ * It's a little bigger than your standard button.
  *
  *      @example
- *      button.btn.btn--blue Click
+ *      button.btn.btn--large Click Me, I'm Bigger!
  */
-@mixin btn--blue {
-  background-color: $blue;
-  border-color: $blue;
+
+
+// A button with white text and a solid colored background
+@mixin btn--solid-color ($background-color: $gray, $hover-color: $darkgray) {
+  background-color: $background-color;
+  border-color: $background-color;
   color: white;
 
   &:not([disabled]) {
     &:hover,
     &:active {
       outline: none;
-      background-color: $darkblue;
-      border-color: $darkblue;
+      background-color: $hover-color;
+      border-color: $hover-color;
       color: white;
       opacity: 1;
     }
   }
 }
 
-/**
- * Large Button:
- * `.btn--large`
- * It's a little bigger than your standard button.
- *
- *      @example
- *      button.btn.btn--large Click Me, I'm Bigger!
- */
+// Button with a larger font size than usual and padding to match
 @mixin btn--large {
   padding: 3px 18px;
   line-height: 36px;
   @include fontsize(16);
 }
+
+// Button with bold text
+@mixin btn--strongtext {
+  font-weight: 600;
+}
+
+// Button with facebook colors
+@mixin btn--facebook {
+  @include btn--solid-color(
+    $background-color: $facebook,
+    $hover-color: $facebook-hover
+    );
+}
+
+// Blue button
+@mixin btn--blue {
+  @include btn--solid-color(
+    $background-color: $blue,
+    $hover-color: $darkblue
+  );
+}
+
+// Button style for login/signup and other account forms
+@mixin btn--account-form {
```

Comment:
I think that this is getting too specific for the buttons mixin. I think that the nypr-accounts-settings repo should import the buttons partial and either create this mixin in that repo or just specify a class that uses these mixins and CSS rules.